src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/UpdateRecallControllerTest.kt

```diff
@@ -68,7 +68,7 @@ class UpdateRecallControllerTest {
     recallEmailReceivedDateTime = fullyPopulatedUpdateRecallRequest.recallEmailReceivedDateTime,
     lastReleasePrison = fullyPopulatedUpdateRecallRequest.lastReleasePrison,
     lastReleaseDate = fullyPopulatedUpdateRecallRequest.lastReleaseDate,
-    localPoliceService = fullyPopulatedUpdateRecallRequest.localPoliceService,
+    localPoliceForce = fullyPopulatedUpdateRecallRequest.localPoliceService,
```

Comment:
We don't want to change this reference to `fullyPopulatedUpdateRecallRequest.localPoliceForce` now?
Obvs it'll happen with the clean-up ticket ...

Reply:
Yeah I did think that initially, but then thought it didn't really matter as this was how it will end up eventually anyway, so slightly less to tidy up!  :-)